lib/block-patterns.php

```diff
@@ -180,8 +156,15 @@ function register_gutenberg_patterns() {
 								<!-- wp:social-link {"url":"#","service":"chain"} /-->
 								<!-- wp:social-link {"url":"#","service":"mail"} /--></ul>
 								<!-- /wp:social-links -->',
-		)
+		),
 	);
+
+	foreach ( $patterns as $name => $pattern ) {
+		$pattern_name = 'core/' . $name;
+		if ( ! WP_Block_Patterns_Registry::get_instance()->is_registered( $pattern_name ) ) {
+			register_block_pattern( $pattern_name, $pattern );
```

Comment:
@youknowriad should we unregister patterns from core if they exist? We do it for blocks.

Reply:
Yes, I think it's better to override them.

Reply:
Done here: https://github.com/WordPress/gutenberg/pull/32086/commits/a38ff60f13d204e9b8d1330f9ce01aaa0af03fc2